js/main.js

```diff
@@ -1,40 +1,29 @@
 /* global data */
 /* exported data */
 
-var entryArray = [];
-var nextEntryId = 0;
 var $photoURL = document.querySelector('#photo');
 var $img = document.querySelector('img');
+if (data.entries.length > 1) {
+  data.nextEntryId = data.entries[0].entryId + 1;
+}
```

Comment:
This `if` on 6-9 shouldn't be necessary. What's it doing for you?

Reply:
if statement has been removed.